spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -141,5 +177,83 @@ module.exports = function (provider, setup) {
     });
   });
 
-  // TODO: add tests for tcpsocket.secure, accepting multiple.
+  it("Secures a socket", function (done) {
+    socket.connect('www.google.com', 80, function () {
+      var prepared = false;
+      dispatch.gotMessageAsync('onData', [], function (msg) {
+        expect(prepared).toEqual(true);
+        socket.secure(done);
+      });
+      // prepareSecure is Chrome specific, see freedom-for-chrome
+      socket.prepareSecure(function () {
+        prepared = true;
+        socket.write(rawStringToBuffer('GET / HTTP/1.0\n\n'),
```

Comment:
I'm confused by this.
we are not actually securing the socket here - google port 80 doesn't have ssl.  Instead you're calling prepareSecure, which won't do anything, except maybe pause the socket, making a non-secure request, and then waiting for data to show up.